server/routes/checkInformationRoutes.ts

```diff
@@ -72,7 +72,7 @@ export default class CheckInformationRoutes {
     const { caseloads, token } = res.locals.user
     const { nomsId } = req.params
 
-    await this.prisonerService.checkPrisonerAccess(nomsId, caseloads, token)
+    await this.prisonerService.checkPrisonerAccess(nomsId, caseloads, token, false)
```

Comment:
I'm confused why this is hardcoded to false rather than checking if the user has the role?